main.go

```diff
@@ -105,6 +105,12 @@ func mainE(ctx context.Context) error {
 
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.Address, "http://127.0.0.1:6443", "Address used to connect to Kubernetes. When empty in-cluster config is created.")
 	daemonCommand.PersistentFlags().Bool(f.Service.Kubernetes.InCluster, false, "Whether to use the in-cluster config to authenticate with Kubernetes.")
+	daemonCommand.PersistentFlags().String(f.Service.Azure.ClientID, "", "ID of the Active Directory Service Principal.")
+	daemonCommand.PersistentFlags().String(f.Service.Azure.ClientSecret, "", "Secret of the Active Directory Service Principal.")
```

Comment:
Uhm, is this passed as a real process `argv` value or via some mounted configuration? I'm afraid of leak via process listing.

Reply:
I don't know. I copied what we have in azure-operator.

Reply:
I'd say mounted https://github.com/giantswarm/azure-disk-mitigator-app/pull/8/files#diff-f070fdc849da3d758a1b7ba914c4b568

Reply:
In `azure-operator` they are mounted, yeah. :+1: